hydroengine_service/dgds_functions.py

```diff
@@ -162,6 +178,12 @@ def visualize_elevation(image,
     :param hillshade: Boolean for hillshading, default True
     :return: Google Earth Engine ee.Image() object, Hillshaded image
     """
+    # validate min is less than max, otherwise raise error
+    min = data_params['bathy_vis_params']['min']
+    max = data_params['topo_vis_params']['max']
+    if min and max is not None:
+        validate_min_lt_max(min, max)
```

Comment:
the function raises an error when min > max. 
Can you send that error message back as a response, that we know what went wrong?

Reply:
the raise error_handler.InvalidUsage will do this for hydroengine (return 400 with message), we should check dgds backend passes this along